int32 type
int32 NOD = 1
int32 HEADBANG = 2
int32 EX_NOD = 3
int32 TILT = 4
int32 SURPRISE = 5
int32 HOLD_SURPRISE = 6
int32 CIRCLE = 7
int32 SHAKE = 8

float64 duration
float64 intensity
float64 rate
float64 delay
bool invert
